### ISSUE TITLE
[BUG] @example + `@` within code example (e.g. decorators) is not parsed correctly.

### ISSUE THREAD

**Issue by shlomiassaf** [compodoc/compodoc#270]
Including `@` sign within at ` ```typescript ` example is not parsed correctly.

```ts
export class Test {

  /**
   * ```typescript
   * @Component({selector: 'someting'}) export class Sometihng {};
   * ```
  value: string;
}
```

I think the parser does not skip special tokens when they are contained (out of scope)

**Comment by vogloblinsky** [comment]
Duplicate of https://github.com/compodoc/compodoc/issues/203

**Comment by vogloblinsky** [comment]
Fixed in 1.1.14
